cosmos/client.go

```diff
@@ -150,10 +150,6 @@ func (c *Client) Stream(ctx context.Context, query string) (chan hash.Hash, chan
 			select {
 			case event := <-eventStream:
 				attrs := event.Events[EventHashType]
-				// The following error might be too much as MAYBE if one transaction contains many messages, the events will be merged across the whole transaction
```

Comment:
have to remove this otherwise services don't receive executions when they are bulk into one transaction